client/src/utils/proxy.js

```diff
@@ -0,0 +1,139 @@
+export default class ProxyDB {
+  constructor(DB_URL) {
+    this.DB_URL = DB_URL;
+  }
+
+  async getData(path = null) {
+    const url = path ? `${this.DB_URL}/${path}` : this.DB_URL;
+    const method = 'GET';
+    let response;
+
+    await fetch(url)
+      .then((res) => {
+        if (!res.ok) {
+          throw Error(`error:${res.status},method:${method},url:${url}`);
+        }
+        return res.json();
+      })
+      .then((data) => {
+        console.log(`${method} Request done successfully`);
+        response = data;
+      })
+      .catch((e) => {
+        console.log(e);
+        response = null;
+      });
+
+    return response;
+  }
+
+  async getDataById(id, path = null) {
+    const url = path ? `${this.DB_URL}/${path}/${id}` : `${this.DB_URL}/${id}`;
+    const method = 'GET';
+    let response;
+
+    await fetch(url)
+      .then((res) => {
+        if (!res.ok) {
+          throw Error(`error:${res.status},method:${method},url:${url}`);
+        }
+        return res.json();
+      })
+      .then((data) => {
+        console.log(`${method} Request done successfully`);
+        response = data;
+      })
+      .catch((e) => {
+        console.log(e);
+        response = null;
+      });
+
+    return response;
+  }
+
+  async post(data, path = null) {
+    const url = path ? `${this.DB_URL}/${path}` : this.DB_URL;
+    const method = 'POST';
+    let response;
+
+    await fetch(url, {
+      method,
+      headers: {
+        'Content-Type': 'application/json',
+      },
```

Comment:
remove the new line here, to be more shortly , 
like : `headers: { 'Content-Type': 'application/json' }`